tests/pipeline/test_alignment.py

```diff
@@ -19,11 +26,19 @@ def test_warp_decam( decam_datastore, decam_reference ):
 
         # Check a couple of spots on the image
         # First, around a star:
+# <<<<<<< HEAD
```

Comment:
Looks like you have a merge conflict baked in here....

...or is this here to remember that at one point we used an abs cutoff rather than a rel, and we're not 100% sure which is what we really want?  If so, probably make the comments reflect that.